src/containers/AppSidebar.tsx

```diff
@@ -4,12 +4,13 @@ import {
   Book,
   Bookmark,
   Folder as FolderIcon,
+  Icon,
```

Comment:
This line was an accident, but I'll be adding it in my next PR anyway :sweat_smile: